src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -0,0 +1,190 @@
+  private static final String DESTINATION_SERVICE_POINT = "inTransitDestinationServicePointId";
+
+  @Test
+  public void reportIsEmptyWhenThereAreNoItemsInTransit()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    List<JsonObject> items = ResourceClient.forItemsInTransitReport(client).getAll();
+
+    assertTrue(items.isEmpty());
+  }
+
+  @Test
+  public void reportIncludeItemInTransit()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource steve = usersFixture.steve();
+    final UUID secondServicePointId = servicePointsFixture.cd2().getId();
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet);
+    createRequest(smallAngryPlanet, steve, secondServicePointId);
+    loansFixture.checkInByBarcode(smallAngryPlanet);
+
+    List<JsonObject> items = ResourceClient.forItemsInTransitReport(client).getAll();
+
+    verifyResponse(smallAngryPlanet, secondServicePointId, items);
+  }
+
+  private void createRequest(InventoryItemResource smallAngryPlanet, IndividualResource steve, UUID secondServicePointId)
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+    RequestBuilder secondRequestBuilderOnItem = new RequestBuilder()
+      .open()
+      .hold()
+      .withPickupServicePointId(secondServicePointId)
+      .forItem(smallAngryPlanet)
+      .by(steve);
+    requestsClient.create(secondRequestBuilderOnItem);
+  }
+
+  private void verifyResponse(InventoryItemResource smallAngryPlanet,
+                              UUID secondServicePointId,
+                              List<JsonObject> items) {
+    JsonObject itemJson = items.get(0);
+    assertThat(itemJson.getString(BARCODE_KEY), is(smallAngryPlanet.getBarcode()));
+    assertThat(itemJson.getJsonObject(STATUS_KEY).getMap().get("name"),
+      is(ItemStatus.IN_TRANSIT.getValue()));
+    assertThat(itemJson.getString(DESTINATION_SERVICE_POINT), is(String.valueOf(secondServicePointId)));
+    final JsonObject smallAngryPlanetInstance = smallAngryPlanet.getInstance().getJson();
+    assertThat(itemJson.getString(TITLE), is(smallAngryPlanetInstance.getString(TITLE)));
+    final String contributors = String.valueOf(((JsonArray) smallAngryPlanetInstance
+      .getMap().get(CONTRIBUTORS)).getJsonObject(0).getMap().get("name"));
+    assertThat(itemJson.getJsonArray(CONTRIBUTORS)
+      .getJsonObject(0).getMap().get("name"), is(contributors));
+    Map<String, String> actualLocation = (Map<String, String>) itemJson.getMap().get("location");
+    assertThat(actualLocation.get(LOCATION_NAME), is("3rd Floor"));
+    assertThat(actualLocation.get(LOCATION_CODE), is("NU/JC/DL/3F"));
+    assertThat(actualLocation.get(LIBRARY), is("Djanogly Learning Resource Centre"));
+  }
```

Comment:
As this method is used by multiple tests, I think it would be better placed at the bottom of the class rather than interspersed with the tests

Reply:
The method takes a list of items as a parameter yet it only checks the first item, why is that?

Reply:
Done